src/raster/r.damflood/SWE.h

```diff
@@ -1,10 +1,10 @@
 
 float velocita_breccia(int i,double h);
 
-/*Funzione per risolvere le shallow water equations
-originariamente sviluppata per r.damflood (GRASS command)
-nel caso generico dare una matrice con 2 raster di 0 **m_DAMBREAK & **m_lake
-e method=3
+/* Function to solve Shallow Water Equations
+Originally developed for r.damflood (GRASS module)
+In the generic case give a matrix with 2 rasters of 0 **m_DAMBREAK & **m_lake
```

Comment:
I'm sure that there is something better than "In the general case", if the Italian subtleties are understood.

Reply:
I think it is correct, but my English is not perfect ;-)